apps/system/utils/fscmd.c

```diff
@@ -133,6 +137,110 @@ static void fscmd_free(FAR char *path)
 		free(path);
 	}
 }
+
+#ifndef CONFIG_DISABLE_ENVIRON
+/****************************************************************************
+ * Name: tash_echo
+ *
+ * Description:
+ *   Display input text or Redirects to a target file
+ *
+ * Usage:
+ *   echo [-n] [input_text] [> or >>] [target path]
+ ****************************************************************************/
+static int tash_echo(int argc, char **args)
+{
+	char *dest_fullpath = NULL;
+	redirection_t direction = { FSCMD_NONE, argc };
+	int i;
+	int fd = 1;
+	int flags;
+	int len;
+	int n_opt = 1;
+	int ret = ERROR;
+
+	if (!strncmp(args[1], "-h", 2)) {
```

Comment:
I found ```--help``` is used to show how to use it. I tried ```cat```, ```cp```. 
1. How about use ```--help``` instead of ```-h```?
2. Linux does not support both ```-h``` and ```--help```  for echo. But I think support is better. Because I mention Linux, I left this.